packages/storefront/src/components/CodeBlock.vue

```diff
@@ -103,6 +103,7 @@
           .token.symbol,
           .token.deleted {
             color: #007299;
+            font-family: Monaco, Menlo, 'Andale Mono', 'Ubuntu Mono', monospace !important;
```

Comment:
Wouldn't defining it once on a higher level, e.g. `code` or `pre` be enough? And why do we need `!important` here?

Reply:
Unfortunately `code` or `pre` ist not enough (the styles were set there already and had no effect on the spans) The important was not needed and I removed it